### PR TITLE
[12.0][MIG] stock_warehouse_calendar

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: stock_warehouse_calendar/__manifest__.py [hunk 1]
```diff
@@ -4,15 +4,15 @@
 {
     "name": "Stock Warehouse Calendar",
     "summary": "Adds a calendar to the Warehouse",
-    "version": "11.0.1.0.0",
-    "license": "AGPL-3",
+    "version": "12.0.1.0.0",
+    "license": "LGPL-3",
```
Comment (rousseldenis) [inline stock_warehouse_calendar/__manifest__.py:8]: @mreficent Is it nomal you change licensing between version ?

**Comment by rousseldenis** [comment]
@mreficent Any news on this ?

File: stock_warehouse_calendar/__manifest__.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply stock_warehouse_calendar/__manifest__.py:8]: I don't know is it's normal or not. But this suppose a problem for you?

Comment (rousseldenis) [reply stock_warehouse_calendar/__manifest__.py:8]: For me no, but I wanted to know if it's wanted on your side to change licensing from AGPL on v11 to LGPL on v12 ?

Comment (MiquelRForgeFlow) [reply stock_warehouse_calendar/__manifest__.py:8]: Yes, it's a wanted thing.

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by HviorForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**